src/EditorFeatures/CSharpTest/AddParameter/AddParameterTests.cs

```diff
@@ -72,7 +72,7 @@ void M()
 @"
 class C
 {
-    public C(int v) { }
+    public C(int @int) { }
```

Comment:
these do not need to be improvements.  I'm actually ok with namign them things like `bool[] bools`.  but `bool @bool` just seems worse than at least the generic `v`.    Can you only make this change for plural values?

Reply:
_"I think these 2 variants are equaly non-descriptive (well, I am not sure, if it is even possible to generate descriptive names in this case not using AI), so there is no need to change code in order to make it work like before"_ - this was in my head, when I was fixing the test. But the idea of making this change only for plurals is realy elegant here.